src/e3/anod/spec.py

```diff
@@ -387,6 +394,101 @@ def bind_to_sandbox(self, sandbox: SandBox) -> None:
             name=self.build_space_name, platform=self.env.platform
         )
 
+    def check_dll_closure(
```

Comment:
dll is windows specific, why not check_shared_library_closure

Reply:
Will do